src/api/app/models/review.rb

```diff
@@ -60,12 +60,16 @@ def state
 
   def accepted_at
     if review_assigned_to && review_assigned_to.state == :accepted
-      review_assigned_to.updated_at
+      review_assigned_to.accept_history_element.created_at
     elsif state == :accepted && !review_assigned_to
-      updated_at
+      accept_history_element.created_at
     end
   end
 
+  def accept_history_element
```

Comment:
Should be in plurarl, and for me `accept` means doing something... Maybe you wanted to say `accepted_history_elements`